app/components/registrations_and_follow_ups_component.rb

```diff
@@ -15,11 +15,7 @@ def initialize(region, current_admin:, repository:, current_period:)
   end
 
   def follow_ups_definition
-    if current_admin.feature_enabled?(:follow_ups_v2)
-      :follow_up_patients_copy_v2
-    else
-      :follow_up_patients_copy
-    end
+    :follow_up_patients_copy
```

Comment:
This method can probably be removed now since the view can simply use `:follow_up_patients_copy`.